common/app/templates/inlineJS/nonBlocking/atomsJs.scala.js

```diff
@@ -0,0 +1,3 @@
+@(page: model.Page)(implicit context: model.ApplicationContext)
+
+@page.metadata.atomsJS.mkString
```

Comment:
why another template with only one line?

Reply:
sorry, this file is very WIP!